.github/workflows/login-quality.yml

```diff
@@ -0,0 +1,68 @@
+name: Login Quality
+
+on:
+  workflow_call:
+    inputs:
+      ignore-run-cache:
+        description: 'Ignore run caches'
+        type: boolean
+        required: true
+
+jobs:
+  quality:
+    name: Ensure Quality
+    runs-on: depot-ubuntu-22.04-8
+    timeout-minutes: 30
+    permissions:
+      id-token: write
+      actions: write
+    env:
+      CACHE_DIR: /tmp/login-run-caches
+    steps:
+      - uses: actions/checkout@v4
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ghcr.io/zitadel/login
+          tags: |
+            type=raw,value=latest,enable={{is_default_branch}}
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+      - uses: depot/setup-action@v1
+        with:
+          oidc: true
+      - name: Restore Run Caches
```

Comment:
Are the caches necessary with the depot runners?

Reply:
the run caches, yes.
These cache the results of docker compose run commands, not build layers.